config/locales/notices/en.yml

```diff
@@ -1,29 +1,28 @@
 en:
   notices:
-    password_changed: Your password was successfully changed.
-    totp_configured: Authenticator app successfully configured
-    totp_disabled: Authenticator app disabled
+    password_changed: You changed your password
```

Comment:
one thing I've noticed is that we are inconsistently using punctuation at the end of notification messages -- do we want to use them or not?